envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/EnvoySnapshotFactory.kt

```diff
@@ -67,7 +68,12 @@ class EnvoySnapshotFactory(
             endpoints = endpoints,
             properties = properties.outgoingPermissions
         )
-        sample.stop(meterRegistry.timer("snapshot-factory.new-snapshot.time"))
+        sample.stop(
+            meterRegistry.timer(
+                "snapshot-factory.seconds",
```

Comment:
maybe snapshot_factory to be consistent with other metrics?

Reply:
yea, I was looking at some our other metrics, but looks like this way would be better, I changed to dots everywhere